src/Fantomas/CodePrinter.fs

```diff
@@ -1439,9 +1439,14 @@ and genExpr astContext synExpr =
         addParenIfAutoNln e1 (genExpr astContext) -- sprintf " <- " +> genExpr astContext e2
 
     | LetOrUseBang(isUse, p, e1, ands, e2) ->
+
+        let genAndList astContext (ands: list<SequencePointInfoForBinding * bool * bool * SynPat * SynExpr * range>) =
+            (sepNone, ands)
+            ||> List.fold (fun fn (_,_,_,pat,expr,_) -> fn +> !- "and! " +> genPat astContext pat -- " = " +> genExpr astContext expr +> sepNln)
```

Comment:
I think you can use the `col` helper function (or a variant) instead of the List.fold.

Reply:
See https://github.com/fsprojects/fantomas/blob/master/src/Fantomas/Context.fs#L283

Reply:
Yep, that worked great! Thanks for the pointer.